requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 click==7.1.2
 dataclasses==0.7; python_version<"3.7"
-neuromation==20.12.7
+neuro-cli>=20.12.11b6
```

Comment:
I would suggest fixating the version to match exactly since previously we had problems with incompatible changes in CLI, which broke neuro-extras.

Also, what about listing both `neuro-cli` and `neuro-sdk` packages in requirements, or it's a bad idea? Are we going to split prepo for those two packages and release them separately in the future?

Everything else LGTM 🙂

Reply:
neuro-sdk and neuro-cli will stay in the same repo for years I think for the sake of development simplification.

Enumeration of both sdk and cli dependencies works but tedious: on every release you'll get two dependabot PRs that should be applied separately: sdk first, wait for master and cli PR rebuilds, than apply cli.

With pip 20.3+ and its refactored dependency resolver listing all second and third level dependencies is not as important as it was before.